frontend/src/components/Dashboard/Dashboard.tsx

```diff
@@ -192,6 +192,7 @@ if(loading){
   );
 };
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
 interface pickupObject {
```

Comment:
Can you change the names of these? The error message means that they want the first letter to be capitalized